components/script/dom/formdata.rs

```diff
@@ -62,7 +62,15 @@ impl FormData {
         form: Option<&HTMLFormElement>,
     ) -> Fallible<DomRoot<FormData>> {
         // TODO: Construct form data set for form if it is supplied
```

Comment:
We can remove this TODO. Also let's add a link to https://xhr.spec.whatwg.org/#dom-formdata.

Reply:
Thanks for checking this one! I was confused if I should remove it or not!